wandb/cli/cli.py

```diff
@@ -244,7 +244,19 @@ def projects(entity, display=True):
     return projects
 
 
-@cli.command(context_settings=CONTEXT, help="Login to Weights & Biases")
+@cli.command(
+    context_settings=CONTEXT,
+    help="""Verifies and stores your API key for authentication with W&B services.
+
+             By default, only store credentials locally without verifying them with the W&B.
+             To verify credentials, set `--verify=True`.
+
+             For server deployments (dedicated cloud or customer-managed instances),
+             specify the host URL using the `--host` flag. You can also set environment
+             variables `WANDB_BASE_URL` and `WANDB_API_KEY` instead of running
+             the `login` command with host parameters.
+             """,
```

Comment:
This is better as a Python docstring on the method instead of inside the `help` argument to `cli.command`. See examples at https://click.palletsprojects.com/en/stable/documentation/

Same feedback on the others!